main.go

```diff
@@ -70,6 +73,7 @@ func main() {
 	flag.BoolVar(&logJSON, "log-json", false, "Set logging to JSON format.")
 	flag.BoolVar(&watchAllNamespaces, "watch-all-namespaces", true,
 		"Watch for custom resources in all namespaces, if set to false it will only watch the runtime namespace.")
+	flag.StringVar(&databasePath, "database-path", "/badger", "Where to store the persistent database of image metadata")
```

Comment:
Better to have this default to something that'll be sensible if you run it in-place, like `/tmp/badger`. For use in a container with a persistent volume, the argument can be patched.

Reply:
Please use the same arg name and location as source-controller https://github.com/fluxcd/source-controller/blob/main/config/manager/deployment.yaml#L43